<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InnerWidget</class>
 <widget class="QWidget" name="InnerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>8</y>
     <width>881</width>
     <height>871</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,17,6">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Original Image</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Output Image</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QVTKWidget" name="qvtkWidgetLeft"/>
    </item>
    <item row="1" column="1">
     <widget class="QVTKWidget" name="qvtkWidgetRight"/>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="grpSelection">
      <property name="styleSheet">
       <string notr="true">QGroupBox#grpSelection { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } 

QGroupBox#grpSelection::title { 
    background-color: transparent;
 } </string>
      </property>
      <property name="title">
       <string>Actions</string>
      </property>
      <widget class="QPushButton" name="btnClearStrokes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Strokes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSaveStrokes">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>The foreground and background pixels will be saved as binary images in X_Foreground.png and X_Background.png where X is specified in the file save diaglog.</string>
       </property>
       <property name="text">
        <string>Save Strokes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPropagate">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Propagate Structure</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLoadMask">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Mask</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radDrawHole">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>106</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw hole</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radDrawPropagationLine">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw propagation line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>59</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Selection</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="btnExtractPatches">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Extract patches</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
